airbyte-integrations/connectors/source-monday/source_monday/graphql_requester.py

```diff
@@ -79,6 +79,28 @@ def _build_query(self, object_name: str, field_schema: dict, **object_arguments)
         arguments = f"({arguments})" if arguments else ""
         fields = ",".join(fields)
 
+        # Essentially, we construct a query based on schema properties; however, some fields in the schema are conditional.
```

Comment:
Can we make it docstring instead of comments?

Reply:
I intended this to be a comment for a specific string because the purpose of this string might not be obvious